src/corehaptics.cs

```diff
@@ -198,7 +198,7 @@ interface ICHHapticAdvancedPatternPlayer {}
 
 	[Mac (10,15), iOS (13,0)]
 	[Protocol]
-	interface CHHapticAdvancedPatternPlayer : CHHapticPatternPlayer {
```

Comment:
This is wrong :)

`protocol CHHapticAdvancedPatternPlayer<CHHapticPatternPlayer>`